torch/csrc/jit/runtime/static/impl.cpp

```diff
@@ -173,15 +171,22 @@ LivenessMap GetLivenessMap(
   // map Values to its creation order in graph (Note: only traverse top-level
   // nodes such that nodes under control-flows are represented by top-level
   // block nodes)
+  int idx = 1;
```

Comment:
Use `size_t` directly? Why are you starting from 1 not 0? Does it matter?

Reply:
0/1 doesn't matter but what do you mean by "Use `size_t` directly? just

`size_t idx = 0` ? if so then sure.